components/database/couchdb-21/couchdb.p5m

```diff
@@ -0,0 +1,1403 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+<transform link path=usr/bin/(.+)     -> default pkg.linted true >
+<transform link path=usr/lib/$(MACH64)/couchdb-21/(.+) -> default pkg.linted true >
+# to avoid ERROR userland.action001.2        32-bit object 'usr/lib/amd64/couchdb-21/lib/couch-2.1.1/priv/couchjs' in 64-bit path
+<transform file path=usr/lib/$(MACH64)/couchdb-21/lib/couch-$(COMPONENT_VERSION)/priv/couchjs -> default pkg.linted true >
+<transform file path=usr/lib/$(MACH64)/couchdb-21/lib/couch-$(COMPONENT_VERSION)/priv/couch(.+) -> set mode 0555 >
+<transform file path=usr/lib/$(MACH64)/couchdb-21/lib/couch-$(COMPONENT_VERSION)/priv/spawnkillable/couch(.+) -> set mode 0555 >
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+depend type=require fmri=runtime/erlang
+depend type=require fmri=runtime/python-27
+depend type=require fmri=library/python/netaddr-27
+depend type=require fmri=database/couchdb-common
+
+link path=usr/lib/$(MACH64)/couchdb-21/etc target=../../../../etc/couchdb-21
+
+dir  path=etc/couchdb-21/local.d owner=couchdb group=couchdb mode=0755
+
+# users should create a link to their preferred logging system
+file files/couchdb-21.logrotate   path=etc/couchdb-21/couchdb-21.logrotate   owner=couchdb group=couchdb mode=0664 preserve=yes
+file files/couchdb-21.logadm.conf path=etc/couchdb-21/couchdb-21.logadm.conf owner=couchdb group=couchdb mode=0664 preserve=yes
+
+file path=etc/couchdb-21/vm.args
```

Comment:
Please, follow layout of other DBMSes. This should have been /etc/couchdb/2.1/...